index.js

```diff
@@ -1,28 +1,62 @@
 const express = require('express')
+const req = require('express/lib/request')
 const app = express()
-const port = 3000
-const data = require('./data.json')
-
-
-
-app.get("/customers", (req, res) => {
-   res.send(data)
+const port = 5006
```

Comment:
```suggestion
const PORT = 5006
```
Please use capital case for constants that do not change.

Reply:
ok